pkg/apis/extensions/v1alpha1/helper/worker.go

```diff
@@ -21,9 +21,20 @@ func ClusterAutoscalerRequired(pools []extensionsv1alpha1.WorkerPool) bool {
 
 // GetMachineDeploymentClusterAutoscalerAnnotations returns a map of annotations with values intended to be used as cluster-autoscaler options for the worker group
 func GetMachineDeploymentClusterAutoscalerAnnotations(caOptions *extensionsv1alpha1.ClusterAutoscalerOptions) map[string]string {
-	var annotations map[string]string
+	annotations := make(map[string]string)
+	caAnnotationsKeys := []string{
+		extensionsv1alpha1.ScaleDownUtilizationThresholdAnnotation,
+		extensionsv1alpha1.ScaleDownGpuUtilizationThresholdAnnotation,
+		extensionsv1alpha1.ScaleDownUnneededTimeAnnotation,
+		extensionsv1alpha1.ScaleDownUnreadyTimeAnnotation,
+		extensionsv1alpha1.MaxNodeProvisionTimeAnnotation,
```

Comment:
You can set the values to an empty string here itself instead of using a new loop.
```suggestion
	caAnnotationsKeys := []string{
		extensionsv1alpha1.ScaleDownUtilizationThresholdAnnotation: "",
		extensionsv1alpha1.ScaleDownGpuUtilizationThresholdAnnotation: "",
		extensionsv1alpha1.ScaleDownUnneededTimeAnnotation: "",
		extensionsv1alpha1.ScaleDownUnreadyTimeAnnotation: "",
		extensionsv1alpha1.MaxNodeProvisionTimeAnnotation: "",
```